### PR TITLE
Adding entrypoint to allow System properties to be passed in

### PR DESCRIPTION
**Pull request by matzew**
Adding entypoint to allow passing in `"-Dsystem=properties"` to the docker run command

### CODE REVIEW

**Comment by matzew** [comment]
@ziccardi mind reviewing ?

File: wildfly/unifiedpush-wildfly-dev/entrypoint.sh [hunk 1]
```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+set -e
+
+# launch wildfly
+exec /opt/jboss/wildfly/bin/standalone.sh -b 0.0.0.0 $@
```
Comment (ziccardi) [inline wildfly/unifiedpush-wildfly-dev/entrypoint.sh]: Shouldn't we use at lease standalone-full.xml ?

**Comment by ziccardi** [comment]
Added a small comment
Other than that, LGTM

File: wildfly/unifiedpush-wildfly-dev/entrypoint.sh (hunk 1 above)
Comment (matzew) [inline wildfly/unifiedpush-wildfly-dev/entrypoint.sh]: yeah, we do that in the docker file :-)

https://github.com/aerogear/dockerfiles/blob/master/wildfly/Dockerfile#L22-L23

we copy our -full config to the normal format ;-)

**Pull request merged**